src/Microsoft.Health.Dicom.Api.UnitTests/Controllers/WorkitemController.AddTests.cs

```diff
@@ -72,7 +72,7 @@ public async Task GivenWorkitemInstanceUid_WhenItAlreadyExists_ThenReturnConflic
                 Arg.Is(_controller.HttpContext.RequestAborted))
             .Returns(new AddWorkitemResponse(WorkitemResponseStatus.Conflict, new Uri("https://www.microsoft.com")));
 
-        ObjectResult result = await _controller.AddAsync() as ObjectResult;
+        ObjectResult result = await _controller.AddAsync(null) as ObjectResult;
```

Comment:
Given that we cannot actually pass `null` values into the controller, this should be a non-`null` value and in the other tests

Reply:
However, we'll still need to `null` check that first value down the call stack (that we presumably already do)